node/src/cli.rs

```diff
@@ -7,7 +7,7 @@ pub struct Cli {
     pub subcommand: Option<Subcommand>,
 
     #[structopt(long, conflicts_with_all = &["chain", "dev"])]
-    pub release: bool,
+    pub testnet: bool,
```

Comment:
Pleas add a doc comment describing the option e.g. `/// Connect to the testnet` or something similar. It will show up in `--help` as the option description.